hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java

```diff
@@ -51,7 +51,7 @@ public record ContractsConfig(
         @ConfigProperty(defaultValue = "5000000") @NetworkProperty long maxNumber,
         // CHAINID returns 295 (0x0127) for mainnet, 296 (0x0128) for testnet, and 297 (0x0129) for previewnet.
         // c.f. https://hips.hedera.com/hip/hip-26 for reference
-        @ConfigProperty(defaultValue = "295") @NetworkProperty int chainId,
+        @ConfigProperty(defaultValue = "298") @NetworkProperty int chainId,
```

Comment:
I think we should have defaults same as mainnet.  I believe 298 is for dev. 
If we have to change tests for this or override to 298 in tests, I think we should.

Reply:
Gotcha, i wasn't sure about this change.